src/utils/socket.js

```diff
@@ -0,0 +1,50 @@
+const app = require("../app");
+const http = require("http");
+const socketIO = require("socket.io");
+
+const server = http.createServer(app);
+
+server.listen(process.env.PORT || 4000, () => {});
+
+const io = socketIO(server, {
+  cors: {
+    origin: "http://localhost:3000",
+  },
+});
+
+const lobbyNamespace = io.of("/lobby");
+lobbyNamespace.on("connection", (socket) => {
+  const currentUser = [];
+  const { user, userProfile } = socket.handshake.query;
+  currentUser.push({ user: user, userProfile: userProfile });
```

Comment:
```suggestion
  currentUser.push({ user, userProfile });
```
`property shorthand`가 가능해 보입니다.
user와 userProfile의 차이는 무엇일까요?